### PR TITLE
Refactor wifi utils in wifi manager

### PR DESCRIPTION
**Pull request by dr-venkman**
Rename wifi_utils.c to wpa_wifi_utils.c, and include in in build only when CONFIG_WPA_SUPPLICANT is enabled. This policy will later be extended to include other <wifi_vendor>_wifi_utils.c files in the build process.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: framework/src/wifi_manager/Kconfig [hunk 1]
```diff
@@ -12,7 +12,7 @@ config WIFI_MANAGER
     ---help---
         Easy APIs for applications to use and control Wi-Fi features
 
-if WIFI_MANAGER
+if WIFI_MANAGER && ARCH_CHIP_S5JT200
```
Comment (sunghan-chang) [inline framework/src/wifi_manager/Kconfig]: What is the purpose of this? Do you want not to use wifi on artik053?

Comment (dr-venkman) [reply framework/src/wifi_manager/Kconfig]: I expected to build supplicant only for s5jt200. But I understand that sourcing the Kconfig does not mean it gets built. So I will revert the changes. Thanks for pointing out.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**